notary-api/.env.example

```
@@ -12,24 +12,13 @@ CACHE_ADAPTER=memory
 NOTARY_SIGNING_SERVICE_URI='http://localhost:9201'
 NOTARY_VALIDATOR_URI='http://localhost:9300'
 TIMEOUT=1000
-
-DATA_EXCHANGE_ADDRESS='0xcc5d6900b473c7fe4dfd542472bad2a3f76b8c24'
 WEB3_PROVIDER='wss://ropsten.infura.io/ws'
```

```suggestion
WEB3_PROVIDER='https://ropsten.infura.io/v3/598681'
```
We could start using `https` instead of `wss` since we updated web3